src/functions-templates/typescript/scheduled-function/{{name}}.ts

```diff
@@ -0,0 +1,9 @@
+import { Handler, schedule } from '@netlify/functions';
+
+export const handler: Handler = schedule("* * * * *", () => {
```

Comment:
I don't think you need to specify the `Handler` type here because it should be inferred from `schedule`.

Reply:
Also, I think we're trying to promote the human-friendly syntax (e.g. `@hourly`) over a cron string. Perhaps we could use that here? (@minivan is this right?)

Reply:
I agree we should make this as human-friendly as possible. I'll update the template with these suggestions.

Reply:
> Also, I think we're trying to promote the human-friendly syntax (e.g. `@hourly`) over a cron string. Perhaps we could use that here? (@minivan is this right?)

Perhaps we could include a comment of example human-friendly cron syntaxes. This way, users can try these out themselves and look up the rest in our documentation (which doesn't exist yet). What do you think?